test/main.js

```diff
@@ -24,6 +26,8 @@ var SelfInsertingView = View.extend({
     render: function () {}
 });
 
+viewCompliance.view(test, ViewSwitcher, { view: new makeTestView()() });
```

Comment:
Seems like this line keeps the tests from running, with the error being `Object prototype may only be an Object or null.`

Here is the stack:

```
create — ampersand-state.js:31
Base — ampersand-state.js:31
View — ampersand-view.js:25
(anonymous function) — main.js:29
```

It seems that a `Object.create()` in AmpersandState is being called with `undefined` as the sole argument (as `this._definition` is undefined).

Can someone confirm that they're getting the same error running tests locally?